windows/src/desktop/kmshell/main/Keyman.Configuration.UI.UfrmStartInstall.dfm

```diff
@@ -14,36 +14,153 @@ object frmStartInstall: TfrmStartInstall
   Font.Style = []
   OldCreateOrder = False
   Position = poScreenCenter
+  OnCreate = FormCreate
   PixelsPerInch = 96
   TextHeight = 13
-  object lblInstallUpdate: TLabel
-    Left = 72
-    Top = 48
-    Width = 180
-    Height = 19
-    Caption = 'Keyman update available.'
-    Font.Charset = DEFAULT_CHARSET
+  object lblUpdateMessage: TLabel
+    Left = 64
+    Top = 89
+    Width = 313
+    Height = 34
+    Caption = 'An update to Keyman has been downloaded and is ready to install.'
+    Font.Charset = ANSI_CHARSET
```

Comment:
Does `ANSI_CHARSET` work even if we have non-roman localizations?

Reply:
Good catch I didn't even notice the IDE had changed it. I changed the font to match the configuration tab font. 
I will change this back.

Reply:
[Font.Charset](https://learn.microsoft.com/en-us/windows/win32/api/wingdi/ns-wingdi-logfontw) seems like it shouldn't matter for Unicode text. It's used mainly for font fallback for non-Unicode text as far as I can tell -- it's certainly very legacy so I don't think it will impact Unicode text.